qb/config.params.sh

```diff
@@ -18,7 +18,7 @@ HAVE_STRIPES=no            # Stripes menu
 HAVE_RUNAHEAD=yes          # Runahead support
 HAVE_SSL=auto              # SSL/mbedtls support
 C89_SSL=no
-HAVE_BUILTINMBEDTLS=yes    # Bake in the mbedtls library
+HAVE_BUILTINMBEDTLS=auto   # Bake in the mbedtls library
```

Comment:
Why are you doing this? We don't want to rely on system libs for this, I'd prefer to just set it to yes instead.